apps/server/lib/lexical/server/code_intelligence/completion.ex

```diff
@@ -186,6 +188,22 @@ defmodule Lexical.Server.CodeIntelligence.Completion do
           Candidate.Struct
         ]
 
+      Env.in_context?(env, :use) ->
+        case result do
+          %{full_name: full_name} ->
+            with_prefix =
+              RemoteControl.Api.modules_with_prefix(
+                env.project,
+                full_name,
+                predicate(&macro_exported?(:"$1", :__using__, 1))
```

Comment:
I like this API.

Reply:
Heh, I'd much rather use a predicate function